scripts/test.js

```diff
@@ -67,65 +67,66 @@ passingTests.forEach(function(passing_test) {
-//     .expectExitCode(1)
-//     .expectErrors([{
-//       message: 'expected \'My AngularJS App\' to equal \'INTENTIONALLY INCORRECT\'',
-//       stacktrace: 'mocha_failure_spec.js:11:20'
-//     }]);
-
-// executor.addCommandlineTest('node built/cli.js spec/errorTest/pluginsFailingConf.js')
-//     .expectExitCode(1)
-//     .expectErrors([
-//       {message: 'Expected true to be false'},
-//       {message: 'from setup'},
-//       {message: 'from postTest passing'},
-//       {message: 'from postTest failing'},
-//       {message: 'from teardown'}
-//     ]);
-
-// executor.addCommandlineTest('node built/cli.js spec/errorTest/slowHttpAndTimeoutConf.js')
+executor.addCommandlineTest('node built/cli.js spec/errorTest/singleFailure.conf.js')
+    .expectExitCode(1)
+    .expectErrors({
+      stackTrace: 'single_failure_spec1.js:5:38'
+    });
+
+// assert timeout works
+executor.addCommandlineTest('node built/cli.js spec/errorTest/timeout.conf.js')
+    .expectExitCode(1)
+    .expectErrors({
+      message: 'Timeout - Async callback was not invoked within timeout ' +
+          'specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.'
+    })
+    .expectTestDuration(0, 1000);
+
+executor.addCommandlineTest('node built/cli.js spec/errorTest/afterLaunchChangesExitCode.conf.js')
+    .expectExitCode(11)
+    .expectErrors({
+      message: 'Expected \'Hiya\' to equal \'INTENTIONALLY INCORRECT\'.'
+    });
+
+executor.addCommandlineTest('node built/cli.js spec/errorTest/multiFailure.conf.js')
+    .expectExitCode(1)
+    .expectErrors([{
+      message: 'Expected \'Hiya\' to equal \'INTENTIONALLY INCORRECT\'.',
+      stacktrace: 'single_failure_spec1.js:5:32'
+    }, {
+      message: 'Expected \'Hiya\' to equal \'INTENTIONALLY INCORRECT\'.',
+      stacktrace: 'single_failure_spec2.js:5:32'
+    }]);
+
+executor.addCommandlineTest('node built/cli.js spec/errorTest/shardedFailure.conf.js')
+    .expectExitCode(1)
+    .expectErrors([{
+      message: 'Expected \'Hiya\' to equal \'INTENTIONALLY INCORRECT\'.',
+      stacktrace: 'single_failure_spec1.js:5:32'
+    }, {
+      message: 'Expected \'Hiya\' to equal \'INTENTIONALLY INCORRECT\'.',
+      stacktrace: 'single_failure_spec2.js:5:32'
+    }]);
+
+executor.addCommandlineTest('node built/cli.js spec/errorTest/mochaFailure.conf.js')
+    .expectExitCode(1)
+    .expectErrors([{
+      message: 'expected \'My AngularJS App\' to equal \'INTENTIONALLY INCORRECT\'',
+      stacktrace: 'mocha_failure_spec.js:11:20'
+    }]);
+
+executor.addCommandlineTest('node built/cli.js spec/errorTest/pluginsFailing.conf.js')
+    .expectExitCode(1)
+    .expectErrors([
+      {message: 'Expected true to be false'},
+      {message: 'from setup'},
+      {message: 'from postTest passing'},
+      {message: 'from postTest failing'},
+      {message: 'from teardown'}
+    ]);
+
+// TODO(selenium4): turn these on when we figure out the correct error message handling.
```

Comment:
We need to talk about these and the following tests commented out. These are UnhandledPromiseRejectionWarnings. I'm not sure where to fix this but maybe for the next PR.